packages-exp/firebase-exp/installations/package.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I think that to ensure these bundles get built, looking at the build config in https://github.com/firebase/firebase-js-sdk/blob/0c607c992527982666d604bc445a1b857f7da66f/packages-exp/firebase-exp/rollup.config.js#L114 it looks like installations and performance need to be added to the `components` list in https://github.com/firebase/firebase-js-sdk/blob/0c607c992527982666d604bc445a1b857f7da66f/packages-exp/firebase-exp/package.json#L59

Can you test out that `yarn build` in `packages-exp/firebase-exp` generates these files?